.github/workflows/release-workflow-dispatcher.yml

```diff
@@ -0,0 +1,266 @@
+name: Release Workflow Dispatcher
+on:
+  workflow_dispatch:
+    inputs:
+      openebs-crds-version:
```

Comment:
Do we need this as input or just set it to openebs version? Can't remember if we keep bumping this or not?

Reply:
Not required anymore. We are setting it to openebs version already. Removed.